libs/sdk-ui-filters/src/RankingFilter/RankingFilterDropdown.tsx

```diff
@@ -38,6 +38,7 @@ export interface IRankingFilterDropdownProps {
     onDropDownItemMouseOver?: (ref: ObjRefInScope) => void;
     onDropDownItemMouseOut?: () => void;
     anchorEl?: EventTarget | string;
+    isMeasureValueFilterUsed?: boolean;
```

Comment:
I would change it to `enableCustomGranularitySelection` or something similar.
cc @yanlipnican

Reply:
Maybe change type to `{ enable: boolean, warningMessage: string }` so you can propagate message from AD and do not presume why it is disabled as there could be other reasons.

Reply:
customGranularitySelection: { enable: boolean; warningMessage: string; } looks much better